sale_tier_validation/models/sale_order.py

```diff
@@ -7,8 +7,8 @@
 class SaleOrder(models.Model):
     _name = "sale.order"
     _inherit = ["sale.order", "tier.validation"]
-    _state_from = ["draft", "sent", "to approve"]
-    _state_to = ["sale", "approved"]
+    _state_from = ["draft", "sent"]
+    _state_to = ["sent", "sale", "done"]
```

Comment:
For not changing the behavior, I think this is what you have to put:

```suggestion
    _state_to = ["sale", "done"]
```

Reply:
> For not changing the behavior, I think this is what you have to put:

Hello @pedrobaeza thank you for your analysis. I was talking about it with other people involved in our project and we think your suggestion is right. I rebased and amended the commit.